docs/content/docs/development/create-table.md

```diff
@@ -296,6 +296,24 @@ Use approach one if you have a large number of filtered queries
 with only `user_id`, and use approach two if you have a large
 number of filtered queries with only `catalog_id`.
 
+The primary key data types currently supported by the table store
+are as follows:
+```
+| TinyIntType
```

Comment:
Can we check it in reverse? For example, users are reminded of errors in array and map. Because it may not be possible to list all types here, such as LocalZoneTimestamp and Time.

Reply:
Good point, I found that the table store has richer data types than hive, I will update to check primary key in reverse